indra/tests/test_ontology.py

```diff
@@ -335,4 +335,15 @@ def test_standardize_hgnc_fplx_mesh_bug():
 
 def test_ido_parents():
     parents = bio_ontology.get_parents('IDO', '0000514')
-    assert ('IDO', '0000509') in parents
\ No newline at end of file
+    assert ('IDO', '0000509') in parents
+
+
+def test_lspci():
+    assert bio_ontology.get_name('LSPCI', '18') == 'Pentane-1,5-Diamine'
+    members = bio_ontology.get_children('LSPCI', 18)
+    # These are some of the members, not all
+    expected_members = {('CAS', '462-94-2'),
+                        ('CHEBI', 'CHEBI:18127'),
+                        ('CHEMBL', 'CHEMBL119296'),
+                        ('PUBCHEM', '273')}
```

Comment:
what about the HMS-lincs ids and the ecommons ones? are they not in TAS?

Reply:
HMS-LINCS IDs do sometimes show up for TAS compounds (not for this one though), I'm not sure about emolecules, that's probably not relevant from an assembly perspective.

Reply:
Agreed